cmd/liqo-controller-manager/main.go

```diff
@@ -212,12 +213,15 @@ func main() {
 	}
 
 	// Setup operators
```

Comment:
The comment is not valid anymore IIUC

Reply:
Sorry, I am not getting your point :(

Reply:
Nothing important. Just if you can update the comment from that generic one.